src/commons/body-message-renderer.jsx

```diff
@@ -3,22 +3,35 @@
  *
  * SPDX-License-Identifier: AGPL-3.0-only
  */
-import React, { useCallback, useEffect, useRef } from 'react';
-import { forEach, reduce } from 'lodash';
+import React, { useCallback, useEffect, useMemo, useRef } from 'react';
+import { forEach, reduce, map } from 'lodash';
 import { useTranslation } from 'react-i18next';
 import { Container, Text } from '@zextras/carbonio-design-system';
 
 const _CI_REGEX = /^<(.*)>$/;
 const _CI_SRC_REGEX = /^cid:(.*)$/;
+const LINK_REGEX = /\bhttps?:\/\/\S+/g;
```

Comment:
this regex doesn't match urls that don't include the protocol (e.g. `mail.zextras.com/zx/team/user-room/.....`),  is this behavior wanted?

Reply:
I'm not sure but I'll confirm from @AlessioCrestani

Reply:
unresolving because i haven't seen any discussion or resolution on this

Reply:
if you need it here you can find the regex for links validation
https://www.regextester.com/93652

Reply:
pinging @AlessioCrestani